sdk/dotnet/Dagger.SDK/GraphQL/GraphQLClient.cs

```diff
@@ -0,0 +1,42 @@
+
+using System.Net.Http.Headers;
+using System.Net.Http.Json;
+using System.Text;
+
+namespace Dagger.SDK.GraphQL;
+
+/// <summary>
+/// GraphQL client for Dagger.
+/// </summary>
+public class GraphQLClient
+{
+    private readonly HttpClient _httpClient;
+
+    public GraphQLClient() : this(Environment.GetEnvironmentVariable("DAGGER_SESSION_PORT")!, Environment.GetEnvironmentVariable("DAGGER_SESSION_TOKEN")!)
```

Comment:
I think our first priority is to get it support module. So I think it's ok for now to support only run under `dagger run`.